client/src/components/FeedbackForm.js

```diff
@@ -2,22 +2,38 @@
 import React, { useState } from "react";
 import "./feedbackForm.css";
 
-const FeedbackForm = ({ student }) => {
-	console.log("student",student);
-	const [feedbackMessage, setFeedbackMessage] = useState("");
+const FeedbackForm = () => {
+	const [feedbackMessage, setFeedbackMessage] = useState({
+		feedback: "",
+		name: "",
+	});
 	const [givenFeedback, setGivenFeedback] = useState([]);
 	const [feedbackList, setFeedbackList] = useState([]);
 
 	const handleSubmit = (e) => {
 		e.preventDefault();
 		setGivenFeedback(feedbackMessage);
-		setFeedbackList([feedbackMessage, ...feedbackList]);
+		setFeedbackList([feedbackMessage.feedback, ...feedbackList]);
 		setFeedbackMessage("");
+		fetch("/api", {
+			method: "POST",
+			mode: "cors",
+			headers: { "Content-type": "application/json",
+			},
+			body:JSON.stringify(
+				feedbackMessage,
+			),
+		})
+			.then((res) => {
+				res.json(); console.log(res);
```

Comment:
@tayfuntayfun What are we doing with the response here?

Reply:
Converting the response into JSON file and console.log for checking purposes. Console.log can be removed.

Reply:
For further clarification, we we make a HTTP request, the server is always looking for a response in order to complete the request.

Reply:
Also, the most important... remember our post process on the backend is sendign a response...this is for reading it on the frontend ;)